src/orch/orch.c

```diff
@@ -1,17 +1,29 @@
 #include "../common/dbus.h"
+#include "../ini/ini.h"
 #include "controller.h"
 #include "opt.h"
 
 #include <stdio.h>
 #include <stdlib.h>
 
+/* A function that takes care of printing the lines of the ini file  */
+static int ini_handler_func(void *user, const char *section, const char *name, const char *value) {
+        printf("[%s] %s = %s\n", section, name, value);
+        return EXIT_FAILURE;
+}
+
 int main(int argc, char *argv[]) {
         fprintf(stdout, "Hello from orchestrator!\n");
 
         int port;
-        get_opts(argc, argv, &port);
+        char *configPath = calloc(1024, sizeof(char));
+
+        int r = get_opts(argc, argv, &port, configPath);
```

Comment:
You don't need to allocate any memory here and "delegate" this to be done in `get_opts` (see other comment) - as long as you ensure freeing the memory. You already free the memory further down below, but a more convenient way to do this is use `_cleanup_free_` from `src/common/memory.h`:
```suggestion
         _cleanup_free_ char *configPath = NULL;

        int r = get_opts(argc, argv, &port, &configPath);
```